spec/populators/organisation_types_populator_spec.rb

```diff
@@ -0,0 +1,17 @@
+require "rails_helper"
+
+RSpec.describe OrganisationTypesPopulator do
+  it { expect(described_class).to respond_to(:call) }
+  it { expect(described_class.new).to respond_to(:call) }
+
+  describe ".call" do
+    subject(:call) { described_class.call }
+
+    before do
+      # clear out existing Organisation Type data - because it is transactional, the seeded data will be restored
+      OrganisationType.all.map(&:destroy!)
```

Comment:
Any reason not to do this instead?
```suggestion
      OrganisationType.destroy_all
```

Reply:
I don't think so, just followed the same pattern as the other populator specs 🙂

Reply:
have changed 👍